model/task/task.go

```diff
@@ -1046,13 +1052,13 @@ func SetTasksScheduledTime(tasks []Task, scheduledTime time.Time) error {
 	return nil
 }
 
-// Removes tasks older than the unscheduable threshold (e.g. one
-// weeks) from the scheduler queue.
+// Removes host tasks older than the unscheduable threshold (e.g. one weeks)
```

Comment:
"host tasks (in contrast with container tasks)"

Reply:
I don't think this needs clarification to contrast with container tasks, because making the host/container task parentheses comment in every single place where the code is doing something host-specific seems excessive. People should generally understand what a task running on a host is.